provider/resource.go

```diff
@@ -45,99 +45,100 @@ const (
-	azureEventGrid             = "EventGrid"             // Event Grid
-	azureEventHub              = "EventHub"              // Event Hub
-	azureMachineLearning       = "MachineLearning"       // Machine Learning Resources
-	azureMaintenance           = "Maintenance"           // Maintenance Resources
-	azureManagedApplication    = "ManagedApplication"    // ManagedApplication
-	azureManagement            = "Management"            // Management Resources
-	azureMaps                  = "Maps"                  // Maps
-	azureMarketPlace           = "Marketplace"           // Marketplace
-	azureMediaServices         = "MediaServices"         // Media Services
-	azureMedia                 = "Media"                 // Media
-	azureMixedReality          = "MixedReality"          // Mixed Reality
-	azureMonitoring            = "Monitoring"            // Metrics/monitoring resources
-	azureMSSQL                 = "MSSql"                 // MS Sql
-	azureMySQL                 = "MySql"                 // MySql
-	azureNetapp                = "NetApp"                // NetApp
-	azureNetwork               = "Network"               // Networking
-	azureNotificationHub       = "NotificationHub"       // Notification Hub
-	azureOperationalInsights   = "OperationalInsights"   // Operational Insights
-	azurePostgresql            = "PostgreSql"            // Postgress SQL
-	azurePolicy                = "Policy"                // Policy
-	azurePowerBi               = "PowerBI"               // PowerBI
-	azureProximity             = "Proximity"             // Proximity
-	azurePrivateDNS            = "PrivateDns"            // Private DNS
-	azurePrivateLink           = "PrivateLink"           // PrivateLink
-	azurePurview               = "Purview"               // Purview
-	azureRecoveryServices      = "RecoveryServices"      // Recovery Services
-	azureRedis                 = "Redis"                 // RedisCache
-	azureRelay                 = "Relay"                 // Relay
-	azureSecurityCenter        = "SecurityCenter"        // Security Center
-	azureSentinel              = "Sentinel"              // Sentinel
-	azureServiceBus            = "ServiceBus"            // ServiceBus
-	azureServiceFabric         = "ServiceFabric"         // Service Fabric
-	azureSearch                = "Search"                // Search
-	azureSignalr               = "SignalR"               // SignalR
-	azureSiteRecovery          = "SiteRecovery"          // SiteRecovery
-	azureSQL                   = "Sql"                   // SQL
-	azureStack                 = "Stack"                 // Stack
-	azureStorage               = "Storage"               // Storage
-	azureStreamAnalytics       = "StreamAnalytics"       // StreamAnalytics
-	azureSynapse               = "Synapse"               // Synapse
-	azureWaf                   = "Waf"                   // WAF
+	activeDirectoryDomainService = "ActiveDirectoryDomainService" // ActiveDirectoryDomainService
```

Comment:
Is it not just "DomainServices" as in https://github.com/Azure/azure-rest-api-specs/tree/master/specification/domainservices/resource-manager ?

Reply:
I think plural `domainservices` would be a better match than `domainservice`